Formula/png++.rb

```diff
@@ -14,7 +14,61 @@ class Pngxx < Formula
 
   depends_on "libpng"
 
+  # Fix strerror_r usage for OS X clang
+  # patch from <https://savannah.nongnu.org/bugs/?46312>
+  patch :DATA
```

Comment:
Also would be nice to get the patch submitted to https://github.com/Homebrew/patches and use the commit reference from there in a `patch do` format. Inline patches behave fairly unpredictably at times, and people tend to make accidental whitespace changes to them on updates.